raiwidgets/raiwidgets/_cohort.py

```diff
@@ -0,0 +1,186 @@
+# Copyright (c) Microsoft Corporation
+# Licensed under the MIT License.
+
+"""Module for defining cohorts in raiwidgets package."""
+
+from typing import Any, List
+
+import pandas as pd
+
+from responsibleai.exceptions import UserConfigValidationException
+
+
+class CohortFilterMethods:
+    """Defines different methods for cohort filters.
+    """
+    GreaterThan = "greater"  # Strictly greater than a value
+    # Greater than or equal to a value
+    GreaterThanEqualTo = "greater and equal"
+    LessThan = "less"  # Strictly lesser than a value
+    LessThanEqualTo = "less and equal"  # Lesser than or equal to a value
+    Equal = "equal"  # Equal to a value
+    Includes = "includes"  # Includes a set of values
+    Excludes = "excludes"  # Excludes a set of values
```

Comment:
I think these constants should live in one python place.  I see them in error analysis code and now I see them here as well. I think it might be better to unify them somehow into one common constants file?

Reply:
see C:\responsible-ai-widgets\erroranalysis\erroranalysis\_internal\constants.py and where they are used

```
class CohortFilterMethods:
    """Cohort filter methods.
    """

    METHOD_INCLUDES = 'includes'
    METHOD_EXCLUDES = 'excludes'
    METHOD_EQUAL = 'equal'
    METHOD_GREATER = 'greater'
    METHOD_LESS = 'less'
    METHOD_LESS_AND_EQUAL = 'less and equal'
    METHOD_GREATER_AND_EQUAL = 'greater and equal'
    METHOD_RANGE = 'in the range of'
```

Reply:
I think this circles back to the question if we should have a common package for cohorts. In case we do, we can condense all the constants for the cohorts in a common place.

I don't want to take dependency on error analysis for this class. I am introducing the ALL and SINGLE_VALUE_METHODS which I think are not going to be used in error analysis. Hence, I intend to keep this class in _cohort.py. I would change the method variables to match the uppercase as above.